src/commands/base-command.mjs

```diff
@@ -99,13 +104,19 @@ export default class BaseCommand extends Command {
         .addOption(new Option('--cwd <cwd>').hideHelp(true))
         .addOption(new Option('-o, --offline').hideHelp(true))
         .addOption(new Option('--auth <token>', 'Netlify auth token').hideHelp(true))
+        .addOption(
+          new Option('--httpProxy [address]', '[DEPRECATED IN V13] Proxy server address to route requests through.')
```

Comment:
Am I right in thinking that this string will not be visible to anyone since we're setting `hideHelp(true)`?

Reply:
correct!